### PR TITLE
Sort a given sequence numbers

### PR DESCRIPTION
**Pull request by giangle-unikie**
The program will sort given numbers, if there is any value that is not a number in the list, the program will print an error message and exit. 
I already fixed all the issues that Milla and Jouni commented

### CODE REVIEW

**Review by milva-unikie — CHANGES_REQUESTED** [review]
The program does not work with negative floats. 

> Enter the numbers separated by space: -1.6
> Error: -1.6 is not integer or float.

File: sort_numbers.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+#This function will sort a given sequence of numbers
+#if value in the list of numbers is not interger or float, program will exit
+def sort_numbers(numbers):
+    for i, value in enumerate(numbers):
+        try:
+            numbers[i] = float(value)
+        except ValueError:
+            print("Error:", value, "is not a valid number.")
+            sys.exit(1)
+    numbers = [num for num in numbers if num >= 0] + [num for num in numbers if num < 0]
```
Comment (milva-unikie) [inline sort_numbers.py]: Can you explain to me what this (line 13) does?

Comment (giangle-unikie) [reply sort_numbers.py]: I slit the numbers into 2 small lists, the first one stores all numbers >=0, and the other stores all numbers <= 0, after that I add 2 small lists to be a new list containing a both positive and negative numbers.

Comment (milva-unikie) [reply sort_numbers.py]: Why? I don't understand why that is needed.

Comment (giangle-unikie) [reply sort_numbers.py]: Ah, yes, it is redundant. I do not need it when doing the sort numbers.

**Comment by jourau** [comment]
$ python3 sort_numbers.py 
Enter the numbers separated by space: -8 1 11 222 2 22
[-8.0, 1.0, 2.0, 11.0, 22.0, 222.0]

**Comment by jourau** [comment]
When input is integers then output should also be, not floats.

**Review by milva-unikie — APPROVED** [review]
Works fine now!

**Comment by jounraun** [comment]
ok for me

**Review by jourau — APPROVED** [review]

**Pull request merged**